superset/views/core.py

```diff
@@ -828,8 +826,7 @@ def shortner(self):
         obj = models.Url(url=url)
         db.session.add(obj)
         db.session.commit()
-        return Response(
-            '/r/{obj.id}'.format(obj=obj),
+        return Response(url_for('R') + '{obj.id}'.format(obj=obj),
```

Comment:
This is not working. We have to replace proper endpoint value in url_for.

Reply:
replaced with older one.